### PR TITLE
[feat] : Allow admins to revoke a team invite (HBE-230)

### PR DESCRIPTION
**Pull request by ankitsridhar16**
Closes # HBE-233

### Description
Currently Admins can view pending invitations for a team, but they can't revoke an invitation. This PR introduces a mutation to revoke a team invite by invite ID that will be used from the admin dashboard.

### Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

### Additional Information

Testing Instructions:
- Create a team and invite users by email
- get pending invitations for the team by team ID
- use `revokeTeamInviteByAdmin` mutation with a invite ID

### CODE REVIEW

File: packages/hoppscotch-backend/src/admin/admin.service.ts [hunk 1]
```diff
@@ -416,4 +417,19 @@ export class AdminService {
...
+   * Revoke a team invite by ID
+   * @param inviteID Team Invite ID
+   * @returns an Either of void or error
+   */
+  async revokeTeamInviteByID(inviteID: string) {
+    const teamInvite = await this.teamInvitationService.revokeInvitation(
+      inviteID,
+    )();
+
+    if (E.isLeft(teamInvite)) return E.left(TEAM_INVITE_NO_INVITE_FOUND);
```
Comment (mirarifhasan) [inline packages/hoppscotch-backend/src/admin/admin.service.ts:431]:
```suggestion
    if (E.isLeft(teamInvite)) return E.left(teamInvite.left);
```

**Review by mirarifhasan — CHANGES_REQUESTED** [review]

File: packages/hoppscotch-backend/src/admin/admin.service.ts (hunk 1 above)
Comment (balub) [reply packages/hoppscotch-backend/src/admin/admin.service.ts:431]: +1

**Review by balub — CHANGES_REQUESTED** [review]
The team-invitation methods used have been modified after the team-invitations refractor, please update to reflect changes.

**Review by mirarifhasan — APPROVED** [review]
LGTM

**Review by mirarifhasan — APPROVED** [review]

**Review by balub — APPROVED** [review]
LGTM 🚀 .

**Pull request merged**